docs/admins.rst

```diff
@@ -594,3 +594,42 @@ container logs. For instance::
 In this example, this build is stuck talking to the docker client (``docker/client.py``).
 
 .. _`config.json`: https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/schemas/config.json
+
+Remote hosts
+------------
+
+Remote hosts are virtual machine hosts that are used by OSBS to build binary images.
```

Comment:
well they don't have to be virtual

Reply:
I know, later in the docs I mentioned multiple choices for the hosts. Removed this reference to VMs though.